.pre-commit-config.yaml

```diff
@@ -122,8 +122,12 @@ repos:
         exclude: |
             (?x)^(
                 .pre-commit-config.yaml$|
+                cobalt/base/unicode/|
+                cobalt/math/|
```

Comment:
Why should this be excluded?

Reply:
Because this is not our code, it's all copyright Chrome and forked from there.